app-modules/authorization/resources/views/login.blade.php

```diff
@@ -36,54 +36,62 @@
     use AdvisingApp\MultifactorAuthentication\Livewire\MultifactorAuthenticationManagement;
 @endphp
 
-<x-filament-panels::page.simple>
-    @if (filament()->hasRegistration())
-        <x-slot name="subheading">
-            {{ __('filament-panels::pages/auth/login.actions.register.before') }}
+<div class="login_form flex w-full flex-col items-center justify-center gap-8 lg:flex-row">
+    <div class="w-full lg:w-1/2 lg:pr-8">
+        @if (filament()->hasRegistration())
+            <x-slot name="subheading">
+                {{ __('filament-panels::pages/auth/login.actions.register.before') }}
 
-            {{ $this->registerAction }}
-        </x-slot>
-    @endif
+                {{ $this->registerAction }}
+            </x-slot>
+        @endif
 
-    <x-filament-panels::form wire:submit="authenticate">
-        @if ($this->needsMfaSetup)
-            <h3 class="flex items-center gap-2 text-lg font-medium">
-                @svg('heroicon-o-question-mark-circle', 'w-6')
-                Multifactor authentication is required for your account.
-            </h3>
-            <p class="text-sm">To finish enabling two factor authentication, scan the following QR code using your
-                phone's authenticator application or enter the setup key and provide the generated OTP code.</p>
-            <div class="mt-3 flex space-x-4">
-                <div>
-                    {!! $this->getMultifactorQrCode() !!}
-                    <p class="pt-2 text-sm">Setup key {{ decrypt($this->user->multifactor_secret) }}</p>
+        <x-filament-panels::form wire:submit="authenticate">
+            @if ($this->needsMfaSetup)
+                <h3 class="flex items-center gap-2 text-lg font-medium">
+                    @svg('heroicon-o-question-mark-circle', 'w-6')
+                    Multifactor authentication is required for your account.
+                </h3>
+                <p class="text-sm">To finish enabling two factor authentication, scan the following QR code using your
+                    phone's authenticator application or enter the setup key and provide the generated OTP code.</p>
+                <div class="mt-3 flex space-x-4">
+                    <div>
+                        {!! $this->getMultifactorQrCode() !!}
+                        <p class="pt-2 text-sm">Setup key {{ decrypt($this->user->multifactor_secret) }}</p>
+                    </div>
                 </div>
-            </div>
+            @endif
+
+            {{ $this->form }}
+
+            <x-filament-panels::form.actions
+                :actions="$this->getCachedFormActions()"
+                :full-width="$this->hasFullWidthFormActions()"
+            />
+            <x-filament-panels::form.actions
+                :actions="$this->getSsoFormActions()"
+                :full-width="$this->hasFullWidthFormActions()"
+            />
```

Comment:
The Action button labels need to be changed to be just "Microsoft" and "Google" and above this if either are available needs to be the wording "or log in with single sign-on" per the designs.